### ISSUE TITLE
Invalid workflow file generated by cmake-init

### ISSUE THREAD

**Issue by fsgmhoward** [friendlyanon/cmake-init#57]
I have set up the project using cmake-init with vcpkg. I made an initial commit after the generation of code but GitHub reported that `ci.yml` generated has syntax error near the vcpkg installation step.

The file generated is here:
https://pastebin.com/gQTA8p43

And the error reported by GitHub action is near line 55:
![image](https://user-images.githubusercontent.com/10681776/169455508-5b211077-d608-4ce8-b421-47ea56bdcaa0.png)

I did a check on the yaml syntax and think it has no issue. Can someone help me out with this?

**Comment by friendlyanon** [comment]
Indeed, the error has been there since 05bfbfbfa2c8130ed3ef1ff1c6d6fa02fc1d1212 (v0.26.0). The error is actually on the next line. The YAML syntax is incorrect. I'll release a fix for this in a bit.

**Comment by friendlyanon** [comment]
Fixed d4a6712646ed8122da0ce21014a0f78a3236dbd9 and released with v0.30.1

If you just want to fix your YAML quickly, then run sed with `'s/\${{ env\.VCPKG_COMMIT }}/"&"/'` on the file.

Thanks for reporting this!
